priv/repo/migrations/20170607142123_create_setting.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.CreateSetting do
+  use Ecto.Migration
+
+  def change do
+    create table(:settings) do
```

Comment:
I think it's a good idea and maybe there's a way of getting the benefit of both ways.
There's a way of using ecto schemas but not necessarily persisting it in a direct mapping, meaning: we can have a schema like the current with parameter fields, types and validation but persist in a key-value way. Database validations are lost but removes the necessity of migration.